CentralUnit/Models/Lap.cs

```diff
@@ -51,6 +36,31 @@ public void AddPenalties(List<PenaltyEvent> penalties)
             this.penalties.AddRange(penalties);
         }
 
+        /// <summary>
+        /// Get lap time in microseconds. Laps with a DNF get a lap time of -1
+        /// Laps with a DSQ get a normally calulcated lap time
```

Comment:
I was unable to find how a DSQ affects the comparison. How does this work?
A DSQ should end up at the bottom of the list, basically like a DNF but with a time (where the time is of no use at all, other than informative for the rider themselves).

Reply:
So the ideal ordering should be DNF > DSQ > normal

Reply:
Correct. 

In the excel-based magic that was used before, DNF and DSQ were a penalty of 10 minutes.